### PR TITLE
Tests for checking image change upon change in options

### LINKED ISSUE #1447: Adding tests for checking image change upon change in options
### Please describe the problem (or idea)

referencing : [this](https://github.com/publiclab/image-sequencer/pull/1432#issuecomment-572149183)

Adding tests for confirming change in image upon change on module option.

### What's your PublicLab.org username?

rishabhshukla1999

### PR DESCRIPTION
**Pull request by blurry-x-face**
Fixes #1447 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below
* [x] Insert-step functionality is working correct as expected.

Tests for the following modules have been added in this PR.
* [x] Blur
* [x] Brightness
* [x] Channel
* [x] Color Temperature
* [x] Colorbar
* [x] Colormap
* [x] Contrast
* [x] Convolution
* [x] Dither
* [x] Dynamic
* [x] Exposure
* [x] Flip Image
* [x] Gamma Correction
* [x] Grid Overlay
* [x] NDVI
* [x] Noise Reduction
* [x] Replace Color
* [x] Rotate
* [x] Saturation
* [x] Threshold
* [x] Tint
* [x] White Balance

### CODE REVIEW

**Comment by blurry-x-face** [comment]
@keshav234156 @jywarren @HarshKhandeparkar What are your thoughts on this?
Should we test the options change this way or do you have any other plan in mind?

**Comment by blurry-x-face** [comment]
@publiclab/is-reviewers

File: test/core/templates/options-test.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+const ImageSequencer = require('../../../src/ImageSequencer');
+
+const red =
+  'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQAQMAAAAlPW0iAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAABlBMVEX+AAD///+KQee0AAAAAWJLR0QB/wIt3gAAAAd0SU1FB+EGHRIVAvrm6EMAAAAMSURBVAjXY2AgDQAAADAAAceqhY4AAAAldEVYdGRhdGU6Y3JlYXRlADIwMTctMDYtMjlUMTg6MjE6MDIrMDI6MDDGD83DAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDE3LTA2LTI5VDE4OjIxOjAyKzAyOjAwt1J1fwAAAABJRU5ErkJggg==';
+target = 'test_outputs';
+
+/**
+ * @method optionsChangeTest
+ * @description a common test for modules
```
Comment (HarshKhandeparkar) [inline test/core/templates/options-test.js]: Please capitalize the first letter and add a full stop at the end of a comment.

File: test/core/templates/options-test.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ * @param {String} moduleName name of the module
+ * @param {'Object'} [options] array of options
+ * @param {String} [benchmark] dataURI of the benchmark images, a image for easch option
+ * @param {String} [input='red_image'] optional input image. Default is a red image.
+ */
+module.exports = (moduleName, options, benchmark, input) => {
+  let sequencer = ImageSequencer({ ui: false });
+
+  test(`${moduleName} module works correctly with different options`, t => {
+    sequencer.loadImages(input || red);
```
Comment (HarshKhandeparkar) [inline test/core/templates/options-test.js]: Could you add some more inline comments describing the code below, to make it easy to understand for future readers.

**Comment by blurry-x-face** [comment]
@HarshKhandeparkar Do you think this is the best way to test this? Or should I look for an alternative?

**Comment by HarshKhandeparkar** [comment]
I guess it is good for now. I don't see any downside to this as of now. (But if we change the modules in the future, it may be a problem)

**Comment by blurry-x-face** [comment]
If we change the modules in future the test will fail and corresponding benchark/options can be changed easily, i don't think that would be a problem.

**Comment by HarshKhandeparkar** [comment]
Great. Please make the requested changes.

File: test/core/templates/options-test.js (hunk 1 above)
Comment (blurry-x-face) [reply test/core/templates/options-test.js]: Done!

File: test/core/templates/options-test.js (hunk 2 above)
Comment (blurry-x-face) [reply test/core/templates/options-test.js]: Done!

**Comment by jywarren** [comment]
Wow, this is really super. Great work, @blurry-x-face !!!

Thanks a ton!

**Pull request merged**